org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/managers/StandardProjectsManager.java

```diff
@@ -262,7 +267,26 @@ public static void configureSettings(Preferences preferences) {
 			}
 		}
 		initializeDefaultOptions(preferences);
+		PreferenceManager.initializeJavaCoreOptions();
```

Comment:
Is there any reason to call this given that InitializeDefaultOptions(..) also ends up calling this ?

Reply:
Fixed.